.gitignore

```diff
@@ -5,6 +5,9 @@ __pycache__/
 # C extensions
 *.so
 
+# .gitignore repo
+joe/data
```

Comment:
I'm not  sure about this, but
if we already have the files in our repo and put `joe/data` in .gitignore, the `data` will remain but the folder won't be tracked in the future.

If the above behavior is true, isn't that favorable for our situation?